src/server/index.tsx

```diff
@@ -123,7 +123,11 @@ export const createServer = ({
     formAction: ["'none'"],
     frameAncestors: ["'none'"],
     frameSrc: ["'none'"],
-    imgSrc: [staticSrc, 'data:'],
+    imgSrc: [
+      staticSrc,
+      'https://reviewers.addons-dev.allizom.org',
```

Comment:
Note that these are just temporary changes, left here to illustrate where I got to in terms of struggling with CSP. I understand that we would never hardcode values like this.